Classes/Controller/ClipboardStateContainer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace Neos\Neos\Ui\Controller;
+
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Error\Messages\Message;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+
+/**
+ * This is a container for clipboard state that needs to be persisted server side
+ *
+ * @Flow\Scope("session")
+ */
+class ClipboardStateContainer
+{
+    /**
+     * @var NodeInterface
+     */
+    protected $clipboardSubject = null;
+
+    /**
+     * @var string
+     */
+    protected $clipboardMode = '';
```

Comment:
What about introducing constants for the mode and tweak this to sth like:
```
/**
 * @var string one of the NodeClipboardInterface::MODE_*  constants
 */
private $mode;
```
to avoid the magic strings